<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Connection</string>
         </attribute>
         <widget class="QLineEdit" name="lEditReadableName">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>170</y>
            <width>211</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>180</x>
            <y>140</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Props</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lEditSendUserName">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>78</y>
            <width>211</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="lEditIpAddr">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>18</y>
            <width>211</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>140.80.61.57</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnUdpatePassword">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>220</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Version</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnLogin">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>108</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>IPAddr</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lEditTextPassword">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>108</y>
            <width>211</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lEditTextUpdatePassword">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>222</y>
            <width>211</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>172</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Nutzername</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>UserName</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>214</y>
            <width>47</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>new Password</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="conResult" native="true">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>30</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lEditTextProperties">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>138</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSetReadableName">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>170</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="lEditTextVersion">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>138</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>47</x>
            <y>200</y>
            <width>321</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Control</string>
         </attribute>
         <widget class="QPushButton" name="btnRefreshControl">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>400</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Statistic</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="txtEditStatistic">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>91</width>
            <height>131</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSaveControl">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>400</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
